portfolio/src/main/webapp/income-expense.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Income vs. Expense</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Income vs. Expense Calculator</h1>
+      <form action="income-expense.html">
+        <input type="file" accept=".csv, text/csv" name="transaction-file"/>
+        <input type="submit"/>
```

Comment:
Can you post a screenshot of what this page looks like?

Reply:
<img width="972" alt="Screen Shot 2022-07-08 at 2 06 34 PM" src="https://user-images.githubusercontent.com/52755071/178070392-c966b1fc-b447-4182-99d6-b7184887292e.png">